cdap-ui/server/express.js

```diff
@@ -112,6 +113,23 @@ function makeApp (authAddress, cdapConfig, uiSettings) {
     res.send('window.CDAP_UI_CONFIG = ' + fileConfig+ ';');
   });
 
+  app.get('/ui-theme.js', function (req, res) {
+    let uiThemeName = uiSettings['ui.theme'] || 'default';
+
+    let path = `${__dirname}/config/themes/${uiThemeName}.json`;
+    try {
+      let fileConfig = JSON.parse(fs.readFileSync(path, 'utf8'));
```

Comment:
You might want to split these into two `try...catch` blocks. One means the file doesn't exists and the other means file exists but has JSON errors. Would be more straight forward if we log different messages for these errors.